packages/database/prisma/schema.prisma

```diff
@@ -246,6 +248,33 @@ model TrackedKeyword {
   @@unique([userId, keyword, providerId]) // Prevents duplicate tracking
 }
 
+model Competitor {
+  id               Int                    @id @default(autoincrement())
+  name             String                 // Company/brand name to track
+  userId           String
+  user             User                   @relation(fields: [userId], references: [id], onDelete: Cascade)
+  isActive         Boolean                @default(true)
+  createdAt        DateTime               @default(now())
+  updatedAt        DateTime               @updatedAt
+  sentimentData    CompetitorSentiment[]
+
+  @@unique([userId, name]) // Prevents duplicate competitor tracking per user
+}
+
+model CompetitorSentiment {
+  id           Int        @id @default(autoincrement())
```

Comment:
When saving competitor sentiment, i wonder if we should think about this data being able to be used by others, in the event someone else is analyszing the same competitor?